CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v5.4.0 - 2025-05-14
+
+### Highlights
+
+Kotlin bindings only: transactions are now [`NonCancellable`](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-non-cancellable/),
+as [required by Uniffi](https://mozilla.github.io/uniffi-rs/latest/futures.html#cancelling-async-code). This prevents a category of bug where Kotlin thinks a transaction
```

Comment:
Nit: `bug` -> `bugs`

Reply:
No, "bugs" would be grammatically improper here.

Reply:
Eh, it seems it's not so straightforward (yay, natural language!), apparently both forms are fine, and it's related to whether you're are referring to a concept, as well as the distinction between countable and uncountable nouns.